README.md

```diff
@@ -1,3827 +1,32 @@
-//Invalid
-/** Invalid */
-/**
- * Invalid
- */
-```
-
-### disallowSemicolons
-
-Disallows lines from ending in a semicolon.
-
-Type: `Boolean`
-
-Value: `true`
-
-#### Example
-
-```js
-"disallowSemicolons": true
-```
-
-##### Valid
-
-```js
-var a = 1
-;[b].forEach(c)
-```
-
-##### Invalid
-
-```js
-var a = 1;
-[b].forEach(c);
-```
-
-### ~~validateJSDoc~~
-
-Please use the [JSCS-JSDoc](https://github.com/jscs-dev/jscs-jsdoc) plugin instead.
-
-### safeContextKeyword
-
-Option to check `var that = this` expressions
-
-Type: `Array` or `String`
-
-Values: String value used for context local declaration
-
-#### Example
-
-```js
-"safeContextKeyword": ["that"]
-```
-
-##### Valid
-
-```js
-var that = this;
-```
-
-##### Invalid
-
-```js
-var _this = this;
-```
-
-## Removed Rules
-
-### ~~disallowLeftStickedOperators~~
-
-Use the following rules instead:
-
-* requireSpaceBeforeBinaryOperators
-* requireSpaceBeforePostfixUnaryOperators
-* requireSpacesInConditionalExpression
-
-### ~~disallowRightStickedOperators~~
-
-Use the following rules instead:
-
-* requireSpaceAfterBinaryOperators
-* requireSpaceAfterPrefixUnaryOperators
-* requireSpacesInConditionalExpression
-
-### ~~requireLeftStickedOperators~~
-
-Use the following rules instead:
-
-* disallowSpaceBeforeBinaryOperators
-* disallowSpaceBeforePostfixUnaryOperators
-* disallowSpacesInConditionalExpression
-
-### ~~requireRightStickedOperators~~
-
-Use the following rules instead:
-
-* disallowSpaceAfterBinaryOperators
-* disallowSpaceAfterPrefixUnaryOperators
-* disallowSpacesInConditionalExpression
-
-## Browser Usage
-
-The npm package contains a file named `jscs-browser.js` (since version `1.5.7`), which is a browser compatible version of `jscs`.
-
-If you'd like to generate this file yourself, run `npm run browserify` after cloning this repo.
-
-Use `jscs-browser.js` on your page as follows:
-
-```html
-<script src="jscs-browser.js"></script>
-<script>
-    var checker = new JscsStringChecker();
-    checker.registerDefaultRules();
-    checker.configure({disallowMultipleVarDecl: true});
-    var errors = checker.checkString('var x, y = 1;');
-    errors.getErrorList().forEach(function(error) {
-        console.log(errors.explainError(error));
-    });
-</script>
-```
+#### More about JSCS
+
+ * [Tutorial and Overview](http://jscs.info/overview.html)
+ * [Codestyle Rules](http://jscs.info/rules.html)
+ * [How to Contribute](http://jscs.info/contributing.html)
+ * [Development Team](http://jscs.info/team.html)
+
+#### Who uses JSCS?
+
+ * [jQuery](http://jquery.com/)
+ * [Adobe](http://www.adobe.com/)
+ * [Twitter Bootstrap](http://getbootstrap.com/)
+ * [AngularJS](https://angularjs.org/)
+ * [Yandex](https://yandex.com/)
+ * [Wikimedia](https://www.wikimedia.org/)
+ * [Grunt](http://gruntjs.com/)
+ * [Ember.js](http://www.emberjs.com/)
+ * [Ionic](http://ionicframework.com/)
+ * [Moment.js](http://momentjs.com/)
+ * [TodoMVC](http://todomvc.com/)
+ * [Famous](http://famo.us/)
```

Comment:
Yeah, I mean something like "JSCS is distributed under MIT License, see LICENSE file" etc.

Almost each repo has it.